content/docs/validation.mdx

```diff
@@ -54,3 +54,21 @@ For example in React it could look like this:
 Here you can see the different modes in action
 
 <ValidationExample />
+
+## External Validation Errors
+
+External errors (e.g. coming from a backend) can be supplied to the form via the `additionalErrors` prop. These errors are mixed in with the regular errors coming from the AJV validation.
+The prop needs to be updated when necessary, as the [ValidationMode](#validationmode) has no effect on the external errors and only affects the errors coming from AJV. 
+
+```js
+const externalErrors: ErrorObject[] = ... ;
+
+<JsonForms
+  schema={schema}
+  uischema={uischema}
+  data={data}
+  renderers={materialRenderers}
+  cells={materialCells}
+  additionalErrors={externalErrors}
```

Comment:
One executable example would be nice, i.e. with an "add/remove external error" button which adds an error which is clearly not from AJV, i.e. with some custom text. Ideally the same error shown here in the code example then.